docs-compose/overwatch_docs/content/DataEngineer/Definitions/071x.md

```diff
@@ -0,0 +1,1005 @@
+| organization_id           | string  | Canonical workspace id                                                                                                                                                                                                    |
+| workspace_name            | string  | Customizable human-legible name of the workspace, should be globally unique within the organization                                                                                                                       |                                                                                                                      
+| warehouse_id              | string  | Canonical workspace warehouse id                                                                                                                                                                                          |                                                                                                                     
+| warehouse_name            | string  | User-defined name of the warehouse                                                                                                                                                                                        |                                                                                                                    
+| service_name              | string  | Name of the service corresponding to DBSQL warehouse                                                                                                                                                                      |                                                                                                                   
+| action_name               | string  | create, edit, or snapImpute – depicts the type of action for the cluster – **snapImpute is used on first run to initialize the state of the cluster even if it wasn’t created/edited since audit logs began               |              
+| user_email                | string  | Email of the user requesting the action                                                                                                                                                                                   |                                                                                                                                                                                  
+| cluster_size              | string  | Size of the clusters allocated for this warehouse. Increasing the size of a spark cluster allows you to run larger queries on it. If you want to increase the number of concurrent queries, please tune max_num_clusters  | 
```

Comment:
I don't think we should give perf. tuning advice here

Reply:
done